torchinfo/torchinfo.py

```diff
@@ -547,6 +549,7 @@ def hook(module: nn.Module, inputs: Any, outputs: Any) -> None:
         info.input_size, _ = info.calculate_size(inputs, batch_dim)
         info.output_size, elem_bytes = info.calculate_size(outputs, batch_dim)
         info.output_bytes = elem_bytes * prod(info.output_size)
+        info.is_trainable = info.get_is_trainable()
```

Comment:
If possible, we should run this in prehook instead, since it does not depend on the forward pass for correctness.